src/state/reducer.ts

```diff
@@ -37,10 +37,21 @@ const appReducer = combineReducers({
 
 export type AppState = ReturnType<typeof appReducer>
 
+function createStorage(dbName: string) {
+  const db = localForage.createInstance({
+    name: dbName,
+  })
+  return {
+    getItem: db.getItem,
+    setItem: db.setItem,
+    removeItem: db.removeItem,
+  }
+}
+
 const persistConfig = {
   key: 'root',
   version: 0, // see migrations.ts for more details about this version
-  storage,
+  storage: createStorage('redux'),
```

Comment:
wait, do we want to move everything out of localStorage? i thought we were just moving lists?

if we are switching everything, we need to modify the initial migration since it assumes we're copying localStorage => localStorage